accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,502 @@
+make it easier to use, she's adding a new `SkipSharpImage` type, which
+represents a bitmap and be constructed via OS-provided data types. Ada uses
+`#if` to expose different constructors on different platforms:
+
+```C#
+public static class SkiaSharpImage
+{
+#if ANDROID
+    public SkiaSharpImage(Android.Media.Image nativeImage) { /* ... */  }
+#endif
+
+#if IOS
+    public SkiaSharpImage(NSImage nativeImage) { /* ... */ }
+#endif
+
+#if WINDOWS
+    public SkiaSharpImage(Windows.Media.BitmapImage nativeImage) { /* ... */ }
+#endif
+}
+```
+
+## Requirements
+
+### Goals
+
+* Use naming that aligns with product strategy
+* Merge .NET Core and .NET Standard into a single concept
+* Developers should be able to understand compatibility relationships without
+  having to consult a mapping table.
+* Provide compatibility with existing concepts and NuGet packages
+* It would be great to get this into .NET 5 Preview 1
+
+### Non-Goals
+
+* Replace TFMs or expand runtime identifiers (RIDs)
+* Support multi-targeting between OS versions
+
+## Design
+
+| TFM             | Compatible With                                            | Comments                          |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+| net5.0          | net1..4 (with NU1701 warning)                              | No WinForms or WPF                |
+|                 | netcoreapp1..3.1 (warning when WinForms/WPF is referenced) |                                   |
+|                 | netstandard1..2.1                                          |                                   |
+| net5.0-android  | xamarin.android                                            |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-ios      | xamarin.ios                                                |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-macos    | xamarin.macos                                              |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-tvos     | xamarin.tvos                                               |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-watchos  | xamarin.watchos                                            |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-win      | netcoreapp1..3.1                                           | WinForms + WPF and probably WinUI |
```

Comment:
The use of the phrase 'probably WinUI' is going to confuse/frustrate some people, as I think you found when you did the video.  You may want to somehow clarify the scenario you are talking about:

- WinUI will probably be supported by .NET 5.0, but it might not be supported at all
OR
- WinUI will definitely be supported by .NET 5.0, and will probably be categorized under the 'net5.0-win' TFM, but it might be categorized under a new TFM not shown here.

Reply:
I've removed WinUI entirely for now to avoid any confusion. Their plans are still up in the air, so which version of .NET will support is speculative at this point.